src/js/component/list/List.js

```diff
@@ -61,13 +61,6 @@ class List extends React.Component {
         });
     }
 
-    fetch() {
```

Comment:
How come this was removed?

Reply:
Strangely enough, after re-adding it's no longer complaining.

Reply:
Huh, interesting. Maybe it is safe to remove. I think as long as the K2 dashboard still loads all the data, it's probably all still working correctly.

Reply:
I think it would be better to remove the unused method instead of disabling the eslint rule for this line.